### PR TITLE
feat: add lightweight verification review

### LINKED ISSUE #42: feat: lightweight verification lacks code review
### 🔎 Existing issue check

- [x] I have searched the existing issues.

### 🧭 Problem

Now:
The lightweight path of comet-verify only performs 5 checks (task completion, file matching, build success, test success, and absence of hardcoded keys). There is no code quality or specification compliance review here.

### ✨ Proposed solution

Suggestion: 
In the lightweight verification process, add item 6: Dispatch a simplified version of the code review sub-agent, which will only check correctness + security + edge cases, without conducting spec/drift checks.

### 🎯 Primary area

Skill workflow

### 🪐 Workflow phase

Verify

### 🔀 Alternatives considered

_No response_

### 🧰 Compatibility notes

_No response_

### 🧩 Additional context

_No response_

### PR DESCRIPTION
**Pull request by ddddddddwp**
## Summary

- Require lightweight verification to include a scoped code review.
- Limit the review to correctness, security, and edge cases.
- Add workflow safeguard coverage and a changelog entry.
- Address the prior review nit by capitalizing the English lightweight review checklist item.

## Test Plan

- `npx vitest run test/ts/skills.test.ts`
- `git diff --check origin/master..HEAD`

## Notes

- Full `npx vitest run` currently fails in `test/ts/init-e2e.test.ts` because the clean-directory init fixture cannot find `.claude/skills` in the temporary Lingma Superpowers staging directory, which also leaves `.codex/skills/comet/SKILL.md` absent.

### CODE REVIEW

**Comment by benym** [comment]
Please fix the issues in the Gemini review first.

**Comment by ddddddddwp** [comment]
已经解决Gemini评论中的问题。

File: assets/skills-zh/comet-verify/SKILL.md [hunk 1]
```diff
@@ -91,8 +91,11 @@ bash "$COMET_STATE" set <change-name> verify_mode full
 3. 编译通过（执行项目对应的构建命令，如 `npm run build`、`mvn compile`、`cargo build` 等）
 4. 相关测试通过
 5. 无明显安全问题（无硬编码密钥、无新增 unsafe 操作）
+6. 简化代码审查通过：派发 code-reviewer 子 agent，只检查正确性、安全、边界条件
 
-**通过标准**：5 项全部 OK，无 CRITICAL 问题。
+简化代码审查的输入应限定为本次改动 diff、tasks.md 和必要的测试结果；审查范围只覆盖实现正确性、安全风险和边界条件，不执行 spec 覆盖率、Design Doc 一致性或漂移检查。若审查发现 CRITICAL 或 IMPORTANT 问题，按验证失败处理并进入 Step 1b。
+
+**通过标准**：6 项全部 OK，无 CRITICAL 问题。
```
[the remark this answers is not in the document]
Comment (ddddddddwp) [reply assets/skills-zh/comet-verify/SKILL.md]: 按照建议已经修复了

File: assets/skills/comet-verify/SKILL.md [hunk 2]
```diff
@@ -91,8 +91,11 @@ When scale assessment result is "small", skip `openspec-verify-change` and direc
 3. Build passes (run project-specific build command, e.g., `npm run build`, `mvn compile`, `cargo build`, etc.)
 4. Related tests pass
 5. No obvious security issues (no hardcoded keys, no new unsafe operations)
+6. Lightweight code review passes: dispatch a code-reviewer subagent that checks only correctness, security, and edge cases
 
-**Pass criteria**: All 5 items OK, no CRITICAL issues.
+The lightweight code review input should be limited to this change's diff, tasks.md, and necessary test results; the review scope covers implementation correctness, security risk, and edge cases only, and does not perform spec coverage, Design Doc consistency, or drift checks. If the review finds CRITICAL or IMPORTANT issues, treat verification as failed and enter Step 1b.
+
+**Pass criteria**: All 6 items OK, no CRITICAL issues.
```
[the remark this answers is not in the document]
Comment (ddddddddwp) [reply assets/skills/comet-verify/SKILL.md]: 按照建议已经修复了

**Comment by benym** [comment]
There appear to be two logical flaws：
1. The lightweight verification process for `/comet-verify` has been expanded to six items; however, the hotfix documentation still refers to a "5-item quick check." Since the hotfix mechanism directly invokes `comet-verify`, this outdated description could potentially cause the agent—when operating under the preset hotfix path—to overlook the newly added "scoped code review" step.
2. A newly introduced rule states that if a code review identifies CRITICAL or IMPORTANT issues, the "lightweight verification" is deemed to have failed; furthermore, the passing criteria explicitly require that there be "no CRITICAL or IMPORTANT issues." However, Step 1b still stipulates that only CRITICAL issues require remediation, while non-CRITICAL failures are considered acceptable deviations.
Consequently—given that IMPORTANT issues fall under the "non-CRITICAL" category—this creates an ambiguity between the "prohibition imposed by the passing criteria" and the "acceptance of deviations permitted by the failure decision logic."

**Comment by benym** [comment]
Please resolve the merge conflict with master first so that we can continue.

**Comment by ddddddddwp** [comment]
> Please resolve the merge conflict with master first so that we can continue.

Already resolved

File: assets/skills-zh/comet-verify/SKILL.md [hunk 3]
```diff
@@ -91,8 +91,11 @@ git diff --stat "$BASE_REF"...HEAD
 3. 编译通过（执行项目对应的构建命令，如 `npm run build`、`mvn compile`、`cargo build` 等）
 4. 相关测试通过
 5. 无明显安全问题（无硬编码密钥、无新增 unsafe 操作）
+6. 简化代码审查通过：派发 code-reviewer 子 agent，只检查正确性、安全、边界条件
```
Comment (benym) [inline assets/skills-zh/comet-verify/SKILL.md:98]: The code-reviewer sub-agent dispatch should be modified to use the Superpowers `requesting-code-review` to dispatch the code reviewer sub-agent, explicitly specifying the triggered Skill.

File: assets/skills/comet-verify/SKILL.md [hunk 4]
```diff
@@ -91,8 +90,11 @@ When scale assessment result is "small", skip `openspec-verify-change` and direc
 3. Build passes (run project-specific build command, e.g., `npm run build`, `mvn compile`, `cargo build`, etc.)
 4. Related tests pass
 5. No obvious security issues (no hardcoded keys, no new unsafe operations)
+6. Lightweight code review passes: dispatch a code-reviewer subagent that checks only correctness, security, and edge cases
```
Comment (benym) [inline assets/skills/comet-verify/SKILL.md:97]: same reason with chinese

**Comment by benym** [comment]
Please fix the issues raised in the review.

**Comment by ddddddddwp** [comment]
> Please fix the issues raised in the review.

都已经确认所有issues已经修复,
